possibly redundant line in ProcessBuildConfig function

jinzhao3611 opened the pull request:
There is a possible redundant line in ProcessBuildConfig function

Commit:
Wrote CreateHandPose.cs for turning hands into shadows of their former selves.

jinzhao3611 committed:
possibly redundant line in ProcessBuildConfig function

keighrim commented on Assets/Editor/StandaloneBuild/Builder.cs:
```diff
@@ -56,10 +56,14 @@ public static class AutoBuilder {
+                            // found a file
+                            // write the name of the scene to ScenesList
+                            // no other path information, no file extension
+                            // jin: sceneName seems to be redundant and probably could be deleted according to the comments above.
```
name will show up in `git blame`, no need to explicitly put names in the code lines.

dyna2016 committed:
commented out sceneName variable

dyna2016 committed:
Revert "commented out sceneName variable"

This reverts commit 9cf3502c5fcd685e1aa138e0f55e9bd368071e74.

jinzhao3611 committed:
Revert "possibly redundant line in ProcessBuildConfig function"

This reverts commit f13c095d390272b665bb8e1a157a6471622fc8a7.

jinzhao3611 committed:
commented out sceneName variable

Commit:
added some warnings to documentation about how the agent will be desrtoyed.

nkrishnaswamy committed:
Added new socket architecture but something's not right with REST connection

nkrishnaswamy committed:
Forgot to add files on last commit

nkrishnaswamy committed:
Merge branch 'spring-19-refactor' into tUESnFiF-mhutchens

nkrishnaswamy committed:
Typo in .gitignore comment

nkrishnaswamy committed:
Fixed Diana mystery hand rotation
Added check to not try to make connections if they are marked inactive
Added OnTargetRotationChanged and OnTargetScaleChanged to Voxeme.cs

nkrishnaswamy committed:
Imported new Standard Assets package

jinzhao3611 committed:
Merge branch 'spring-19-refactor' into 54sXtk1m-jin
up-to-date version of Standard Assets

nkrishnaswamy committed:
Added new put.xml file for VoxML interpretation from file

jinzhao3611 committed:
Merge branch 'spring-19-refactor' into 54sXtk1m-jin
updated Standard Assets Package

jinzhao3611 committed:
commented out extraneous line
